datalad/support/annexrepo.py

```diff
@@ -386,8 +395,8 @@ def annex_enableremote(self, name):
 
         self._run_annex_command('enableremote', annex_options=[name])
 
-    @normalize_paths
-    def annex_addurl_to_file(self, file, url, options=[]):
+    @normalize_path
+    def annex_addurl_to_file(self, file_, url, options=[]):
```

Comment:
`file_` instead of `file` since it is a builtin keyword till somewhere in python3, and lintians complaint for overloading it ;-)